ui/components/app/alerts-system/multiple-alert-modal/multiple-alert-modal.tsx

```diff
@@ -144,32 +146,46 @@ export function MultipleAlertModal({
   onFinalAcknowledgeClick,
   ownerId,
 }: MultipleAlertModalProps) {
-  const { alerts } = useAlerts(ownerId);
+  const { isAlertConfirmed, fieldAlerts: alerts } = useAlerts(ownerId);
 
   const [selectedIndex, setSelectedIndex] = useState(
-    alerts.findIndex((alert) => alert.key === alertKey),
+    alerts.findIndex((alert: Alert) => alert.key === alertKey),
```

Comment:
Minor, but why do we need to define `Alert` here since won't `alerts` always be an `Alert[]` since it would be empty rather than undefined in the worst case?

Reply:
I just included it cause Lint was complaining. I think there were more places where it started to point that `implicitly has an 'any' type`.